src/serializer/serializer.js

```diff
@@ -1252,8 +1253,10 @@ export class Serializer {
           let serializedBinding: SerializedBinding = serializedBindings[name];
           if (serializedBinding.modified && !serializedBinding.referentialized) {
             let serializedBindingId = t.identifier(this.referentializedNameGenerator.generate(name));
+            let serializedValue = serializedBinding.serializedValue;
+            invariant(serializedValue);
```

Comment:
I was unable to convince myself that this invariant is true. I trust that it is, but just from a quick reading of the code it is not obvious to me.